proxy/round_tripper/proxy_round_tripper.go

```diff
@@ -191,6 +191,7 @@ func (rt *roundTripper) RoundTrip(originalRequest *http.Request) (*http.Response
 				slog.Float64("dns-lookup-time", trace.DnsTime()),
 				slog.Float64("dial-time", trace.DialTime()),
 				slog.Float64("tls-handshake-time", trace.TlsTime()),
+				slog.String("local-addr", trace.LocalAddr()),
```

Comment:
❓ Why call it "local-addr" here, but "local_address" above? Why not "local-address" here?

Reply:
Fixed it.